cmake/pcre2-config.cmake.in

```diff
@@ -105,41 +80,5 @@ endif()
 unset(PCRE2_8BIT_COMPONENT)
 unset(PCRE2_POSIX_COMPONENT)
 
-include(FindPackageHandleStandardArgs)
-set(${CMAKE_FIND_PACKAGE_NAME}_CONFIG "${CMAKE_CURRENT_LIST_FILE}")
-find_package_handle_standard_args(PCRE2
-  FOUND_VAR PCRE2_FOUND
-  REQUIRED_VARS PCRE2_INCLUDE_DIR
-  HANDLE_COMPONENTS
-  VERSION_VAR PCRE2_VERSION
-  CONFIG_MODE
-)
-
-set(PCRE2_LIBRARIES)
-if (PCRE2_FOUND)
-  foreach(component ${PCRE2_FIND_COMPONENTS})
-    if (PCRE2_USE_STATIC_LIBS)
-      add_library(PCRE2::${component} STATIC IMPORTED)
-      target_compile_definitions(PCRE2::${component} INTERFACE PCRE2_STATIC)
-    else ()
-      add_library(PCRE2::${component} SHARED IMPORTED)
-    endif ()
-    set_target_properties(PCRE2::${component} PROPERTIES
-      IMPORTED_LOCATION "${PCRE2_${component}_LIBRARY}"
-      INTERFACE_INCLUDE_DIRECTORIES "${PCRE2_INCLUDE_DIR}"
-    )
-    if (component STREQUAL "POSIX")
-      set_target_properties(PCRE2::${component} PROPERTIES
-        INTERFACE_LINK_LIBRARIES "PCRE2::8BIT"
-        LINK_LIBRARIES "PCRE2::8BIT"
-      )
-    endif ()
-
-    set(PCRE2_LIBRARIES ${PCRE2_LIBRARIES} ${PCRE2_${component}_LIBRARY})
```

Comment:
Here, you are removing the output variable interface. While it is legacy CMake (in contrast to targets), it is unclear if some users still rely on it. 
However, it is also unclear which variables form the public interface. IMO it is at least: 
`PCRE2_LIBRARIES` (subject to components)
`PCRE2_INCLUDE_DIR`
maybe also `PCRE2_${component}_LIBRARY`.
I would recommend to at least place the target names in the lib variables. (This still leaves room for incompatibilities but common cases would be covered.)

Reply:
I set the library variables to the library paths.